modules/ROOT/pages/whats-new-in-dw.adoc

```diff
@@ -0,0 +1,72 @@
+= What's New in DataWeave
+
+The 2.4 version of DataWeave introduces the following features:
+
+* Larger than memory strings.
+* New reader and writer properties for working with data formats.
+* Many new modules, functions, types, annotations, and variables. Some are experimental and subject to change in future releases.
+* A number of other overloaded functions introduce new helper functions for
+handling `null` values.
+
+Changes to the documentation include more detailed function signatures and organization of the functions by their module names, such as `dw::Core`, `dw::core::Strings`, and dw::util::Math.
+
+DataWeave 2.4 is bundled with Mule 4.4. For information about the Mule 4.4 release, see xref:4.4@mule-runtime::whats-new-in-mule.adoc[What's New in Mule 4.4].
+
+== New Data Formats and Properties
+
+DataWeave introduces the following reader and writer properties:
+
+* DataWeave (dw) reader property: `privileges`
```

Comment:
Should we add a brief description of these?

Reply:
TODO: privileges

Reply:
```suggestion
* DataWeave (dw) reader property: `privileges` enables you to provide a comma-separated list of privileges to use in the format, such as `'Resources,Properties'`.
```

Reply:
LGTM